src/components/pages/common/resource-info.tsx

```diff
@@ -113,34 +121,45 @@ function CarouselResourceInfo({ currentResource, licensesList }: CarouselResourc
 
               <Box>{t("common:uploader")}</Box>
               <Box>
-                <Link href={`/user/show/${resource?.userIbp?.id}`}>
-                  <ExternalBlueLink>{resource?.userIbp?.name}</ExternalBlueLink>
+                <Link href={`/user/show/${resource?.resourceData?.userIbp?.id}`}>
+                  <ExternalBlueLink>{resource?.resourceData?.userIbp?.name}</ExternalBlueLink>
                 </Link>
               </Box>
 
               {resource?.resource?.uploadTime && (
                 <>
                   <Box>{t("common:resource.upload_time")}</Box>
-                  <Box>{formatTimeStampFromUTC(resource?.resource?.uploadTime)}</Box>
+                  <Box>{formatTimeStampFromUTC(resource?.resourceData?.resource?.uploadTime)}</Box>
                 </>
               )}
 
               <Box>{t("common:resource.license")}</Box>
               <FieldShow
                 field="license"
-                fieldValue={resource?.license?.name}
+                fieldValue={resource?.resourceData?.license?.name}
                 updateFunc={axEditResource}
                 setFetch={setFetch}
                 defaultValue={resource}
                 licensesList={licensesList}
                 canEdit={canEdit}
               />
 
+              <Box>{t("common:media_gallery.title")}</Box>
+              <FieldShow
+                field="mediaGallery"
+                updateFunc={axEditResource}
+                setFetch={setFetch}
```

Comment:
can you rename setFetch to something a little more descriptive and meaningful?

Reply:
resolved .